src/modules/task/components/TaskTable/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Table } from 'antd';
+import dayjs from 'dayjs';
+import TaskProgressBar from '../TaskProgressBar';
+import {
+  InprogressPhoto,
+  TotalPhoto,
+  ElaspedTime,
+  Active,
+} from 'Modules/task/mocks/data';
+import { ModelEnum } from 'Modules/task/models/model.enum';
+import { stringCutter } from 'Modules/history/utils/stringCutter';
+import { TaskTableWrapper } from './styled';
+
+export type TaskTablePropsType = {
+  data: [
+    {
+      key: string;
+      timestamp: string;
+      taskID: string;
+      taskName: string;
+      model: ModelEnum;
+      progress: number;
+      elaspedTime: string;
+      inprogressPhoto: number;
+      totalPhoto: number;
+      active: boolean;
+    },
+  ][];
+};
+
+const TaskTable: React.FC<TaskTablePropsType> = (props) => {
+  const { data } = props;
+  const { Column } = Table;
+  return (
+    <TaskTableWrapper dataSource={data}>
+      <Column
+        title="Timestamp"
+        dataIndex="timestamp"
+        key="timestamp"
+        render={(timestamp: Date): JSX.Element => {
+          return (
+            <div>
+              {dayjs(timestamp).format('YYYY-MM-DD')}
```

Comment:
`YYYY-MM-DD ` and `HH:mm:ss ` should define it as a constant variable for reusable and clearly format. becuase of `MM` and `mm` will make it confuse to me which one is Month or minutes.
example.
```ts
const YEAR_MONTH_DATE_FORMAT = 'YYYY-MM-DD'
```

Reply:
fixed at 28f9c300967302c9c93236c58a3b6c4daf3e3573